etc/release/before-liquibase.sh

```diff
@@ -18,15 +18,186 @@
+export INSTALL_URL_ADMIN
+export INSTALL_URL_PROJECT
+
 #The following parameter needed for Postgres, if our current repo is embedded in to another repo. 
 # The default CREATE_DATABASE is initialized in step1, and needs to be overwritten during runtime.
 # If you need to embed it, then CREATE_DATABASE=false, and with this, we can avoid to create an empty DB within the "parent (host)" DB. 
 # This parameter is used in "dookug-db/liquibase/common/_create_dbs/create-database.sql"
 CREATE_DATABASE=${CREATE_DATABASE:-true}
 echo "CREATE_DATABASE=$CREATE_DATABASE"
 export CREATE_DATABASE
+
+# The following parameter is used to set the order of the install steps.
+# The default order is 1,2,3,4.
+# If you want to alter the order, or you just need specific step(s), you can set the INSTALL_STEPS environment variable outside from DevOps.
+# The variable cannot be empty! If all steps are required, then set it to "1,2,3,4, etc.".
+STEPS=${INSTALL_STEPS:-"1,2,3,4"}
+echo "INSTALL_STEPS=$STEPS"
+echo "---------------------------"
+IFS=',' read -ra STEPS <<< "$STEPS"
+
+#sanity check:
+#creating an ordered copy.
+SORTED_STEPS=$(printf '%s\n' "${STEPS[@]}" | sort -n | paste -sd "," -)
+#if the steps are not in order.
+if [[ "$STEPS" != "$SORTED_STEPS" ]]; then
+  # order it by ascending order.
+  SORTED_STEPS=$(printf '%s\n' "${STEPS[@]}" | sort -n | paste -sd "," -)
+  # overwrite the STEPS variable with the ordered one.
+  IFS=',' read -ra STEPS <<< "$SORTED_STEPS"
+fi  
+
+# ----------Postgres install------------------
+if [[ "$AUTO_INSTALL" == "postgresql" ]]; then
+  for step in "${STEPS[@]}"; do
+    case $step in
+      1)
+        echo "Running step-01 - DB initialization..."
+        echo "  INSTALL_USERNAME_ADMIN=$INSTALL_USERNAME_ADMIN"
+        echo "  INSTALL_URL_ADMIN=$INSTALL_URL_ADMIN"
+        liquibase --defaults-file=./postgresql/defaults-step-01.properties \
+                  --changeLogFile=/changelog/liquibase-install-step-01.xml \
+                  --url="${INSTALL_URL_ADMIN}" \
+                  --username="${INSTALL_USERNAME_ADMIN}" \
+                  --password="${INSTALL_PASSWORD_ADMIN}" \
+                  update
+        ;;
+      2)
+        if [[ "$INSTALL_PGTOOLS" == "1" ]]; then         
+          echo "Running step-02 - ${S2_SCHEMA_NAME} pg_tools installation..."
+          echo "  INSTALL_USERNAME_PROJECT=$INSTALL_USERNAME_PROJECT"
+          echo "  INSTALL_URL_PROJECT=$INSTALL_URL_PROJECT"
+          liquibase --searchPath=../${DOCKER_REPOSITORY}/liquibase/changelog \
+              --defaults-file=./postgresql/defaults-step-02.properties \
+              --changeLogFile=./partman/liquibase/changelog/liquibase-install-step-02.xml \
+              --url="${INSTALL_URL_PROJECT}" \
+              --username="${INSTALL_USERNAME_PROJECT}" \
+              --password="${INSTALL_PASSWORD_PROJECT}" \
+              update
+        fi      
+
+        echo "Running step-02 - ${S2_SCHEMA_NAME} objects creation..."
```

Comment:
schema név egyértelműsítésre pl (a többi helyen is):
```suggestion
        echo "Running step-02 - Objects creation into ${S2_SCHEMA_NAME} schema..."
```

Reply:
Javítottam